generator/src/main/java/com/scottlogic/deg/generator/walker/reductive/ReductiveState.java

```diff
@@ -27,6 +27,10 @@ private ReductiveState(
         this.lastFixedField = lastFixedField;
     }
 
+    public boolean allValuesAreFixed() {
+        return this.lastFixedField.hasValueSet() && this.fixedFields.size() == this.fields.size() - 1;
```

Comment:
Can `lastFixedField` be null?

Previously the _head_ (`lastFixedField`) could be null when the state was initialised.